src/Files.Launcher/MessageHandlers/FileOperationsHandler.cs

```diff
@@ -227,8 +314,19 @@ private async Task ParseFileOperationAsync(PipeStream connection, Dictionary<str
 
                                 for (var i = 0; i < fileToDeletePath.Length; i++)
                                 {
-                                    using var shi = new ShellItem(fileToDeletePath[i]);
-                                    op.QueueDeleteOperation(shi);
+                                    if (!Extensions.IgnoreExceptions(() =>
+                                    {
+                                        using var shi = new ShellItem(fileToDeletePath[i]);
+                                        op.QueueDeleteOperation(shi);
+                                    }))
+                                    {
+                                        shellOperationResult.Items.Add(new ShellOperationItemResult()
+                                        {
+                                            Succeeded = false,
+                                            Source = fileToDeletePath[i],
+                                            HRresult = (int)-1
```

Comment:
Unrelated, but Should be Hresult or HResult

Reply:
Thanks lol never noticed